TAInformation/models.py

```diff
@@ -40,3 +40,17 @@ class LabCourseJunction(models.Model):
 class LabTAJunction(models.Model):
     lab_id = models.ForeignKey(Lab, on_delete=models.CASCADE)
     user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+
+
+class courseName(models.Model):
```

Comment:
We have course_name in Course() class already. I'm not sure what purpose this class would have

Reply:
I created the course_name method to use as a  foreign key in the TAassignment method

Reply:
I don't think we can add methods like that to models but I haven't worked much with foreign keys. I'd say lets wait for someone else to review it as well